manim/utils/config_utils.py

```diff
@@ -373,24 +373,24 @@ def _run_config():
     config_files = _paths_config_file()
     if _from_command_line():
         args = _parse_cli(sys.argv[1:])
-        if args.config_file is not None:
-            if os.path.exists(args.config_file):
-                config_files.append(args.config_file)
-            else:
-                raise FileNotFoundError(f"Config file {args.config_file} doesn't exist")
-        else:
-            script_directory_file_config = os.path.join(
-                os.path.dirname(args.file), "manim.cfg"
-            )
-            if os.path.exists(script_directory_file_config):
-                config_files.append(script_directory_file_config)
-
     else:
         # In this case, we still need an empty args object.
         args = _parse_cli([], input=False)
         # Need to populate the options left out
         args.file, args.scene_names, args.output_file = "", "", ""
 
+    if args.config_file is not None:
+        if os.path.exists(args.config_file):
+            config_files.append(args.config_file)
+        else:
+            raise FileNotFoundError(f"Config file {args.config_file} doesn't exist")
+    else:
+        script_directory_file_config = os.path.join(
+            os.path.dirname(args.file), "manim.cfg"
```

Comment:
Won't this cause issues ? If this is ran not from command line, `args.file` won't exist, no ?